grails-app/controllers/org/pih/warehouse/api/ApiController.groovy

```diff
@@ -155,7 +155,8 @@ class ApiController {
         String minimumExpirationDate = dateFormat.format(grailsApplication.config.openboxes.expirationDate.minValue)
         def logoLabel = grailsApplication.config.openboxes.logo.label
         def pageSize = grailsApplication.config.openboxes.api.pagination.pageSize
-        def logoUrl = location?.logo ? "/openboxes/location/viewLogo/${session.warehouse?.id}" : null
+        def logoUrl = location?.logo ? "/openboxes/location/viewLogo/${session.warehouse?.id}" : grailsApplication.config.openboxes.logo.url
+        def defaultLogoUrl = grailsApplication.config.openboxes.logo.url
```

Comment:
it was needed in case `location.logo` exists and the url is "correct", but the image that this url contains does not load, we want to make sure, that we always have access to `defaultLogoUrl` from config, because if we don't have this `defaultLogoUrl`, we only return default logo, if the `location.logo` is null/falsy

Reply:
Since we have this logo in the data base, I think there should be no problem with not loading it.